### PR TITLE
rbspy 0.1.8 (new formula)

### PR DESCRIPTION
**Pull request by jacobbednarz**
Creates a new formula for `rbspy` which is a Ruby profiler.

cc @jvns 

---

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

### CODE REVIEW

File: Formula/rbspy.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class Rbspy < Formula
+  desc "Sampling profiler for Ruby"
+  homepage "https://github.com/rbspy/rbspy"
+  url "https://github.com/rbspy/rbspy/archive/v0.1.8.tar.gz"
+  sha256 "87f759acd9d660178737b9f24cc07f0113a81a9fffc1604bae2c756d7f4d815b"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "build"
```
Comment (jvns) [inline Formula/rbspy.rb]: should be `cargo build --release` (to build a release version instead of a debug version)

File: Formula/rbspy.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    bin.install "target/debug/rbspy"
+  end
+
+  test do
+    output = shell_output("#{bin}/rbspy -V")
```
Comment (jacobbednarz) [inline Formula/rbspy.rb]: I know this is frowned upon however `rbspy` requires `sudo` on mac to perform most of the operations which I something I don't think we want to introduce into a CI test so I settled for the next best thing. I have raised rbspy/rbspy#88 which might help us get a non-root bit of output here.

File: Formula/rbspy.rb (hunk 1 above)
Comment (jacobbednarz) [reply Formula/rbspy.rb]: 🙇 thanks for the heads up! Addressed this one via 41b3e164

**Comment by ilovezfs** [comment]
According to its [README](https://github.com/rbspy/rbspy/blob/f645348a8563be82ee38292051016cc921788792/README.md#rbspy), this is "currently alpha software." In Homebrew, we only package software that upstream considers to be "stable," so unfortunately we'll need to pass on this. You, or upstream, may want to host it in [your own tap](https://github.com/Homebrew/brew/blob/master/docs/How-to-Create-and-Maintain-a-Tap.md#how-to-create-and-maintain-a-tap), though.

Thanks for the PR nonetheless @jacobbednarz

**Comment by jvns** [comment]
@ilovezfs thanks! i agree it's too early to package rbspy. Will submit a PR when it's stable.

**Pull request closed without merging**